### PR TITLE
Alt and friends

### PR DESCRIPTION
**Pull request by rpominov**
See https://github.com/fantasyland/fantasy-land/pull/197

@safareli could you take a look?

### CODE REVIEW

**Comment by safareli** [comment]
Will do tomorrow 🍭

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "lobot": "0.1.22"
   },
   "dependencies": {
-    "fantasy-land": "1.0.1"
+    "fantasy-land": "fantasyland/fantasy-land"
```
Comment (davidchambers) [inline package.json]: v2.1.0 is now available. :)

Comment (rpominov) [inline package.json]: Thanks!

**Comment by safareli** [comment]
Looks good

**Comment by rpominov** [comment]
Great, thank you for taking a look!

**Comment by safareli** [comment]
Welcome!

**Pull request merged**